common/src/test/resources/testPayment.properties

```diff
@@ -0,0 +1,10 @@
+# needed for convinience to make parsing easier and more independed of parameters from outside
+methods.interface=test
+
+# comma separated list of available payment method IDs
+methods.available=test-methodid
+
+# config for each method
+methods.test-methodid.method=test-methodid
```

Comment:
:+1: 

I'll take care of that once the whole architecture is done. Change should not have any impact on the architecture itself.

created issues for that: #11